### DISCUSSION TITLE
I can't upload documents for students developer pack

### DISCUSSION THREAD

**Discussion by ahmetqunay** [community/community#87487] [GitHub Education] [answered]
### Hi everyone,

Hello, I want to continue my Github student membership, but I cannot upload documents in the system. There is only a photo taking section. When I try to take a photo, it only uses the front camera of the phone in reverse. I cannot switch to the rear camera and take it from there. I do not understand why we are not allowed to upload documents. I would be glad if you can help me.
![Ekran görüntüsü 2024-01-15 143334](https://github.com/community/community/assets/97553177/d7225c5f-1f55-424a-a01a-f3ac8bd1f663)

**Comment by anuragjain-git** [comment]
I cannot switch to the rear camera either

**Reply by ahmetqunay** [reply]
How did you solve the problem?

**Reply by Rae99** [reply]
I guess this issue is still unresolved since I ran into it today. I couldn't find anywhere to upload a file or switch to back camera either... And my application got rejected. I opened the digital documents on my mac and tried to take a photo with the front camera, but the photo is of such poor quality...

**Reply by luxkovacs** [reply]
One year later, this is still an issue; I can't use any of my rear cameras either...

**Reply by lusabo** [reply]
Me neither

**Reply by Jemmy778** [reply]
me too

**Reply by JavMB** [reply]
The GitHub Educacion verification process is unnecessarily frustrating! Forcing the use of front-facing camera while other cameras are blocked/downgraded, together with the extreme image quality degradation, makes it impossible complete verification. This appears deliberatly bad, and that is really a horrible way to treat students!. Either the problem is fixed, or I request an alternative validation process

**Reply by syed-akhlaq** [reply]
is there any option available where we can upload the document .because on capturing from the camera even from the printed hardcopy of the document its getting blurred and the quality becomes very poor and its not detected and is constantly rejected

**Reply by abgardn2** [reply]
This is still an issue. No way of directly contacting support, no way of uploading a file or taking a screenshot. Clearly they don't actually want students to be approved. The fact that a transcript or official enrollment letter is "Fair" proof of status is such bogus.

**Comment by BeniNobel** [comment]
There may be a button that says `Upload a file`. I had the same problem, but now I fixed it. Try looking closely or try other links, and you will get one where there's that `Button` for uploading files.

**Reply by Elysium420** [reply]
what type of other links its so frustrating :/ only webcam options how am i supposed to upload

**Comment by mishmanners** [comment]
Thanks for reaching out. I've tagged the appropriate labels and passed this onto the Education team.

**Reply by teressacodes** [reply]
I'm having this issue as well. I've tried on website/laptop and on iphone browser. There is no upload button.

**Reply by Abdullah-222** [reply]
I'm also facing same issues for a couple of months github haven't fixed it till now there's no upload option i can find

**Reply by Rae99** [reply]
Actually in my case, the upload button just showed up somehow once I was physically at school.

**Reply by luxkovacs** [reply]
> Actually in my case, the upload button just showed up somehow once I was physically at school.

I'm assuming I'm in the minority here, but I'm with a distance learning institute, so if that's what it takes for the option to upload to appear ... That's kind of disappointing. Really hope GitHub comes up with something for the likes of myself. (Happy you got it to work for yourself, though!)

**Reply by NEVERdieNM** [reply]
A year later, I have the same issue.

**Reply by abozer** [reply]
Still the same issue

**Comment by samus-aran** [comment, marked as answer]
Hi @ahmetqunay  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Regarding to the use of the rear camera can we ask what browser and operating system? 

Let us know here if you need further assistance

**Reply by ahmetqunay** [reply]
Thank you for your response, but it wasn't helpful at all. The instructions you provided are already in the system, but they are not practical. If it really worked, we wouldn't be opening a request here. I doubt you have ever tried taking a photo using this method. If you want to help us, please introduce the capability to upload documents. I kindly request that from you, please.

**Reply by samus-aran** [reply]
Hi @ahmetqunay 

First off I want to thank you for your candid response and that we absolutely understand your frustration. I know it may feel that information is being repeated but we need to ensure that users have explored these avenues before investigating if there is a specific issue with the platform itself. 

We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value. This is why sometimes we may require a little more evidence or actions from the student applying and we thank you for being patient.

Let's start off with the upload button. @BeniNobel provided a good comment below 👇 We would prefer users to be either on campus or close unless they (and their evidence) states they are enrolled as remote students. Now if you are not able to be on campus (or close by) then my suggestion above provides additional information on how to supply the required evidence without the upload button. 

Hope this helps but let me know if after you've submitted information and need guidance on your rejection reasons 👍

**Reply by ahmetqunay** [reply]
I am studying remotely, I mentioned that I am having difficulty taking photos of documents. Is there another option I can apply, or will you continue to advise me to follow your recommendations and upload the photos?

**Reply by Patrick-Dunn-Dagg** [reply]
Yeah, it still makes no sense why a photo is required rather than the upload if you are a distance student. It would seem likely that most distance students are having their correspondence emailed to them, so the logical step would be uploading the emailed document rather than trying to take a photograph of a screen or having to print it.

**Reply by RosembergAraujo** [reply]
Same problem here, i’m also studying remotely, and all the documents are in PDF. Do i really have to print them out just to take a photo? It’s absurd. My campus is in another city, so i’d have to travel just to take a few photos?

**Reply by bofethe** [reply]
still a problem

**Reply by ghost** [reply]
still a problem

**Reply by Xylobyte** [reply]
still a problem !!!!

**Reply by PokkzDev** [reply]
I've been rejected twice, I even asked my school to forward me a certificate of me studying there. Which of course is a PDF. How am I supposed to take a photo of it if its blurry and insta rejected?

**Reply by StevenLeonCooper** [reply]
> Hi @ahmetqunay 👋
> 
> We're happy you're here!
> 
> It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.
> 
> 1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
> 2. If you can, print off your evidence and take a photo of this copy.
> 3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
> 
> Regarding to the use of the rear camera can we ask what browser and operating system?
> 
> Let us know here if you need further assistance

This is an absolutely ridiculous response. I have the same issue and I am ON CAMPUS CONNECTED TO THE NETWORK ON A COMPUTER. Why would a photo be preferable over a file upload? You expect me to print out my appointment letter and hold it on front of my laptop's crummy camera? This isn't some high school senior project, this is GITHUB. With such an asinine system what would EVER possess me to even continue applying for these "benefits" anyway? No thanks. Your team obviously is insane to begin with; I will not even bother.

**Reply by AntonioFranciulli** [reply]
This is still an ongoing issue that, from what one can see in the comments, its still affecting thousands of students globally. The webcam approach is far from ideal, specially for colleges that don't provide physical IDs for students and the only means of validating ones enrolment are PDFs signed by the authorities.

**Reply by JJKW1984** [reply]
@samus-aran This isn't a helpful answer and it seems to ignore the input from all of us that have a legitimate issue with the application process. Is it not possible to send the documentation required to be processed? It is clear that taking a single camera shot of the information isn't working well for many students. Is there a support team we can email information too ?

**Reply by tdhampson12** [reply]
I submitted a screenshot essentially of like five different documents, and that worked. The camera method is completely broken

**Reply by Lu1zLuna** [reply]
> I submitted a screenshot essentially of like five different documents, and that worked. The camera method is completely broken

How did you submitted a screenshot?? I can only see the option to take use the camera.

**Reply by tdhampson12** [reply]
I think there was an option to select "upload other document" and it wasn't recommended. you also NEED to be on campus, no VPN

**Reply by ExusLT** [reply]
Your answer is pathetic.

**Reply by piplus2** [reply]
Do you know that some proofs are on an A4 page, so they cannot be captured from a close distance in the small frame you provide?

**Reply by MrZombie-11** [reply]
@tdhampson12  where ???? help me please

**Reply by rpages2020** [reply]
I Ttied 20 times with human readable "photos" and rejected me.

**Reply by twistedtrident1** [reply]
you know this thread makes me want to drop github, pull all my research and extinguish them from my future searches. adding an upload is just a few lines of code and ai can read the text on these documents but they would rather discourage a student who is already at his/her brink by rejecting their application 15 or 20 times when they have a verified .edu address.

**Comment by BeniNobel** [comment]
I had the same problem too, but somehow after I reopened the link again, I found the button for `Upload files` and it worked correctly. Also, the first time I did it, It detected that I wasn't on campus and then, after doing it on campus I found the button for **uploading files**. Look if there are any requirements you are not satisfying and then you will find where the problem is.

**Reply by ahmetqunay** [reply]
I am studying remotely, is it logical to have a requirement to be on campus to upload documents? Also, my camera not being good is already a challenge, and on top of that, it flips the photos the wrong way.

**Reply by Mohamed-Aziz-Yahia** [reply]
@BeniNobel @ahmetqunay  By campus you mean your university? If so, can you tell what's the differece between registering from campus and remotly?

**Reply by BeniNobel** [reply]
That might be an issue for `GitHub` to solve. As they check your location to grant you the access, they should provide other ways for remote students to prove that they are truly studying remotely.

**Comment by mateuszwyszynski** [comment]
I have a similar problem. The upload button is there, but after I click "Process my application" nothing happens. I know it says to wait a few seconds, but I actually tried waiting for several minutes and nothing has changed. I tried reopening the tab again as well as cancelling my application and trying again. I'm on a MacOS. I also tried doing this through both Chrome and Safari.

Basically it looks like in the following screenshot and nothing is happening:

<img width="466" alt="Screenshot 2024-01-19 at 11 12 48" src="https://github.com/community/community/assets/30074645/aeb071c2-d0df-418e-9536-d440c330a4a6">

EDIT:

I ended up taking a photo through the interface and submitting an application with this photo. However, I'm not sure what will be the result, because:
- the photo has much worse quality than the file I have prepared
- my student ID has an expiration date on the back, so it's impossible for me to upload a single photo which contains both my personal details and the ID validity date.

**Reply by oskarniziol** [reply]
Hi, I have the same date problem as you, did you manage to solve it?

**Reply by Mohamed-Aziz-Yahia** [reply]
Well, my application got approved but didn't get my benefits yet. The only thing I could do was to accept the location access from the browser and then took a picture with the front camera, and because the quality is bad, I am afraid to get denied.

**Reply by oskarniziol** [reply]
Okay, but the expire date is on the back.

**Reply by Mohamed-Aziz-Yahia** [reply]
what do u mean?

**Reply by oskarniziol** [reply]
on the front, there's a names, issue date, etc. The expirtation date is on the back.

**Reply by mateuszwyszynski** [reply]
@OskiBoski483 no. I sent an application with just the front picture, same as you did. I'm still waiting for an answer.

**Reply by Meff1u** [reply]
any updates?

**Comment by flopmeister** [comment]
Might be a bug on the backend. I'm getting a 413 trying to upload my file (which is between the provided range of 100KB and 10MB). Regardless, there should probably be a visual queue to indicate there's been an error.
![Screen Shot 2024-01-19 at 9 09 48 PM](https://github.com/community/community/assets/19997916/e19199de-fb23-426c-b9d6-ccf97ab619ea)

**Comment by Mohamed-Aziz-Yahia** [comment]
Yeah actually I've got my benefits days later.
I've also updated my situation in GitHub, I don't know if you saw it but I
mentioned that it was solved.
Thanks.

**Comment by NewVu2** [comment]
@samus-aran I have a question im only laptop and my university card have 2 faces front and back on back face the expire date mentioned and on front face my picture issue date and my id.

Secondly how can I upload card image from my laptop I not want to show my face on front camera how can i do this please help me!

**Comment by nickemma** [comment]
This is frustrating. I am studying remotely at university of the people. And I've everything emailed to me. How do you expect me to take a rear picture it doesn't make sense

**Reply by MuckerQA** [reply]
still same issue happening

**Comment by hmperccami** [comment]
Any updates?

**Comment by i-contemplator** [comment]
I can't switch to back camera to take a picture from my phone and on my mac, it is extremely painful to take a clear picture.

**Comment by i-contemplator** [comment]
This is annoying. How is there no better system in place here.

**Comment by bitdruid** [comment]
wow. such a basic functionality simply messed up. on my phone i cant switch to rear camera and on on both - phone / pc i cant upload only select camera... thanks man

update: i took the photo with front camera but it wont focus. so i received an instant decline from github because its not sharp enough. they are joking about their own mess...

**Comment by MrCoolAndroid** [comment]
I'm having issues with this as well, is the support team even looking at this? I can't even upload files!

**Reply by nithin0927** [reply]
How is this still an issue? Is there any solution for this?

**Comment by pratikgondkar** [comment]
+1

**Comment by Bulani** [comment]
I'm having issues with this as well, is the support team even looking at this? I can't even upload files!

**Reply by mirzaabdullaht** [reply]
same here.

**Comment by silverrainb** [comment]
<img width="728" alt="Screenshot 2024-10-30 at 10 05 00" src="https://github.com/user-attachments/assets/e79f8e56-b70e-4dac-8b25-14cecb8b320a">
If you don't provide "upload" button, how am I supposed to upload the document? You want me to physically print and take a photo? When I attempted with my 2nd laptop, it refused to accept the evidence. For months there hasn't been resolution? What gives?

**Reply by mirzaabdullaht** [reply]
Same here

**Reply by abh2050** [reply]
@samus-aran github? Why are you making this complicated? Is there a way you will accept it anyother way? your camera sucks and not accepting it for 3 times in the row.

**Reply by AtomyTR** [reply]
Same issue here.

**Comment by tomislavhofman** [comment]
This is plain stupid for multitude of reasons.

For example I live 3km off of college and I am filling the form from home. Because guess what. I am busy studying at campus, where unlike in the USA, we do not bring laptops with us to school.

**Comment by Law78** [comment]
This system is designed to be unusable. All other websites accept applications from a university email. My University, [IUL](https://lms.iuline.it/), that's present in your list, is an online University; why do I need to justify being far from the campus if it's an online University? This application makes no sense. It’s unclear why I have to use a front camera (the rear one isn't activated), and I can't upload any files showing my payment of fees or enrollment. Essentially, there’s no way to submit documentation that proves my enrollment. Please take inspiration from many other sites that streamline the process simply by using a university email!

**Comment by ChenxiangWang** [comment]
if on interest for providing edu package, it is fine to delete the page instead.

**Comment by btvd** [comment]
Did anyone solve the issue with documents uploads? I have the same issue - can do only photo, but it was rejected without translation, which can not be done with only camera....

**Comment by mirzaabdullaht** [comment]
<img width="215" alt="image" src="https://github.com/user-attachments/assets/61d61459-50d0-4a54-9f7c-cea86375ebe7">

**Hey everyone**, 
                            I wanted to share my experience with getting the GitHub Student Developer Pack. I ran into the same problem many of you are facing - there was no option to upload documents, and when I tried using the live camera, my application kept getting rejected.

**Here's what I did:**
1. I tried applying from different devices
2. I emailed GitHub's customer support
3. I even used a MacBook because its camera is way better than other laptops

My student ID was a bit tricky because it had two pages - one with my name and the other with the expiry date. After a bunch of rejections and following their suggestions, I gave it another shot with the MacBook. It took 5 rejections over a few days, but on my 6th try, it finally worked!

Now I'm just waiting for the benefits to kick in, which should take about 3 days.

**Some tips from my experience:**
- Don't give up. Keep trying until you get approved.
- Make sure to apply before you graduate. I only had 2 months left on my student ID.
- Try to get the clearest picture possible of your ID.

I know how frustrating it is when you're trying to upload a clear document but can't, and you keep getting rejected. But hang in there!

My GitHub education request finally got approved. 

Good luck to all of you trying to get your GitHub Education Pack :)

**Reply by Dianak14** [reply]
how did you contact GitHub customer support? I tried and am getting a popup saying 'individual manual reviews and status updates are no longer provided'.

**Reply by luxkovacs** [reply]
Yeah, same here. Don't get why they're making it so impossible to get some help, especially with the system being as dysfunctional as it is.

**Comment by Focze-Eszter** [comment]
@samus-aran 
How am I supposed to upload the ID if my ID has 2 sides? The UX is really terrible. I'm forced to upload only 1 photo, but in reality I need to upload two, because the identification is on one side and the valability on the other. Your maximum image size is 1MB, BUT you require it to be really a really clear image. Guess what, a clear photo an a modern smartphone is more than 1MB! So I need to manually compress the image. Your logic is not the brightest one. How do I upload the 2 sides of my ID??? I tried to make a collage photo with the 2 sides and it got rejected because "it was manipulated". I tried to use the laptop's camera. Not a solution (Do I need to use a mirror if only one photo is allowed and I have 2 sides?). I also have the ID on the phone, on the university app. I tried to photo that too with the laptop's camera. It looks so unclear, even if my laptop's camera is decent. And also, how am I supposed to send a translation of the documents if only 1 photo is allowed? The logic beats me.
Tell me what should I do? Do I need to email these photos to you guys, send a letter, send a carrier pigeon, or what? Give me a solution and resolve this horrible UX. Is it normal to waste a few months with Github only to get that JetBrain student pack? 

What should you guys do:

1. Allow multiple uploads or live photo's uploads.
2. Change the 1MB limit to 5MB or more.
3. You can also add a live video option, to show all of my documents and translations.
4. Maybe an e-mail, easy to find on that page, to be able to attach multiple documents without the size limit.
5. Maybe a text input for us to give some explanations about the documents.

Come on, guys. This is a beginner mistake. You can do way better.

**Comment by ElairaMeow** [comment]
Having this issue. It was really awkward trying to take a photo of my course registrations with the front camera and still have it be legible. Every other site I've used my student information on to be able to qualify for certain benefits has let me upload then information and it's automatically approved. This is the last site I'd expect to have some broken or missing features.

**Comment by nqtri** [comment]
Same issue here. This is really lame

**Comment by mustfaunlu** [comment]
Same issue😠 . Maybe it's time to finally put your Copilot to work to solve this problem, after all, it’s been pretending to work for years!

**Comment by Rae99** [comment]
I am an international student and currently in my home country. They mentioned this “You appear not to be near any campus location for the school you have selected” in the email, and I thought maybe this is the reason my application won't pass. I am gonna give it another try when I go back to the country my school is located in, but will that really solve the problem? Will I still be unable to upload a file? Are you guys trying to upload files in a different country too, or it won't work either way?

**Reply by mustfaunlu** [reply]
The file upload issue persists and is unrelated to the school location. This problem lies within the website itself.

**Comment by keshavprasadneupane** [comment]
I am having same issue here . can anyone help me I am trying to upload my collage issued id but the system is not registering . My pc has bad camera but when i try using my phone then it wont let me change my camera to back camera{since my phone front camera is a gives very blurry photo } how i am able to upload the id card it keep say "Please provide reason why your are not on campus?" . I was literally uploading this photo from my collage, on my student uniform. how can i do this thing, i am preety much bugged. And now feeling like while student developer pack is a total scam

**Reply by Rae99** [reply]
Yes, they say even if you are right at school, the issue still remains coz it lies in the system regardless of what your location is. I haven't followed this for a long time but right now I'm waiting for my flight to the country I study in. I don't know if I will be able to make it work when I get there. I'm gonna give it a try anyway. I think it's a long shot though. I still don't understand why they refuse to put in some extra work to make the system more user-friendly.

**Reply by Rae99** [reply]
> What can users do if they are not on campus but need to upload their documents?

Sorry, I couldn't provide any useful advice because for me it only worked when I was on campus. Good luck with your application.

**Comment by bofethe** [comment]
Does anyone on the GitHub team monitor this anymore? This still doesn't work.

**Comment by Abdullah-222** [comment]
@samus-aran it's still not working I'm also facing the same issue can't find the upload button although I'm sitting on my campus and applying for it

**Comment by francesco-oghabi** [comment]
Still an issue

**Comment by Dianak14** [comment]
I have been trying the same thing and with only front camera allowed, the pictures of my proof of enrolment are incredibly blurry. The aforementioned 'upload' button to actually upload files has never appeared for me. I am getting instant reject emails when I try to take photos of my documents that are perfectly legible in their pdf format

**Comment by bofethe** [comment]
In my case, it wasn't my application that was the issue, but rather my account settings.  I had to add a 2FA and billing account info (no payment info required) and it finally took even with a super blurry student verification form I took with my webcam while holding up a screenshot from my phone.

**Comment by M-ZABIULLAH** [comment]
Sometimes GitHub might accept other forms of verification if the camera or photo upload isn't functioning as expected. Be sure to check their [GitHub Student Developer Pack FAQs](https://education.github.com/pack) for any additional details or alternative options.

**Comment by msciencesport** [comment]
I have the same proble. Unsable to verify.

**Comment by williammarget** [comment]
[Retention for WordPress ](https://digimarketbuzz.com/customer-retention-for-wordpress/) focuses on keeping users engaged and returning to your site. To enhance retention, implement strategies like personalized content, regular updates, and engaging newsletters. Utilize plugins that encourage user interaction, such as comment sections and forums. Offering loyalty programs or exclusive content can also incentivize repeat visits. Monitor user behavior through analytics to identify drop-off points and improve the user experience. By fostering a community and consistently delivering value, you can significantly increase user retention and satisfaction on your WordPress site.

**Comment by matrodsilver** [comment]
Seriously, what justifies only the front camera being usable? It makes absolutely no sense at all. They want a clear picture but provide no means for it. Still a problem!

**Reply by matrodsilver** [reply]
Word of advice, after a lot of attempts, I was able to do it by not taking a picture of the whole document. Instead of taking a picture of the whole page, I just got the middle part with my name and the signature on it. I'd recommend taking pictures of relevant parts of the document, and hope that you find the right one.

**Comment by wallisson-tech** [comment]
I have the same problem, how to solve this folks.

**Reply by bitdruid** [reply]
> I have the same problem, how to solve this folks.

no solution bro. they managed to sit out this issue for a long time. at least for me they finally managed to make this all a big dump because i'm getting exmatriculation in ~ 3 month - finished my colloqium.

**Comment by rmcavalcante7** [comment]
It’s been practically impossible to take an acceptable photo using the front camera. It’s ridiculous that GitHub is adopting this kind of validation. What’s the problem with attaching an official document issued by the university?

**Comment by joshtype** [comment]
Was accepted at previous remote uni (there was a dedicated portal with upload tool), rejected the next year (portal replaced by a popup tool that records location and launches front-facing camera; tried a photo of a document using laptop front camera), then rejected again at my current remote uni (tried a phone pic of my laptop screen). The last attempt did recognize the document, but I was informed that, as my location was not near my campus, I needed something that proved I was registered for online classes. Interestingly, the popup menu still references "upload of screenshot" as an example of a poor-quality form of proof, while not offering any upload functionality. I think this issue is not resolved because it's useful for limiting the number of free benefits. I've also wondered if I was accepted because I was a freshman and the idea is to offer a free taste to new students to entice them to pay when courses become more advanced.

**Reply by PhiPhiPham** [reply]
OMG THIS!! YOU SAID IT PERFECTLY!! Why on golly goodness name am I supposed to provide proof that I was registered for online classes when there is no upload function. Seriously, like cmon github. On top of that, they are not even checking it manually, so what are they even specifically looking for?!?

**Comment by i-love-parrots** [comment]
Scanned my student ID, it won't let me upload it for some reason even though the size is correct. Got rejected for some stupid reason. Why is the process so unnecessarily onerous? Do I need to give you my social security number, too?

**Comment by tdhampson12** [comment]
Still a problem

**Comment by YadgarovFayzullo** [comment]
My campus is located in another city, and I finished the first year of studying and now im in on vacation in my hometown. It's a stupid idea to upload files only when u in the uni campus

**Comment by wishuuu** [comment]
Will these problems be adressed in any time soon? Process is flawed for quite some time. Previously (when it was possible to drag and drop images), some automatic evaluation kept rejecting my photo, because it was "edited". All I did, was scanning both sides of by student ID and pasted them next to each other in gimp, because my identification is on one side, while date confirming I'm still student on the other. Now it's even worse, because on pc without camera I cannot upload it at all.

**Comment by MrPancetita** [comment]
This process is broken. With an enrollment letter it is impossible to make it work. -2

**Comment by AntonioFranciulli** [comment]
This is still an ongoing issue that, from what one can see in the comments, its still affecting thousands of students globally. The webcam approach is far from ideal, specially for colleges that don't provide physical IDs for students and the only means of validating ones enrolment are PDFs signed by the authorities.

**Comment by Tomask4** [comment]
Still an issue

**Comment by RuiGuilherme** [comment]
You can use OBS Studio; simply add an image as a source and start a virtual camera. This way, GitHub will only see the image you added in OBS.

**Reply by RFBomb** [reply]
That is such a bullshit workaround lmao. 
I was trying to register via the mobile site, but i cant unless i decide to wait until later to print out a paper or take a screenshot, send it to someone else and take their phone. 

What a joke of a verification system. Let us upload PDFs or screenshots instead of photos.

edit: took a screen cap of a pdf and uploaded it from my pc, and it worked fine.

**Comment by Djamth** [comment]
Tengo el mismo problema , no puedo verificar

**Comment by Losidyne** [comment]
Github doesnt give a F. Stupid camera ... scam
